src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+         class="card__image" 
+         src="images/imac.jpeg" 
+         alt="iMac A1419"
+      >
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <div class="card__code">Код товара: 195434</div>
+      <div class="card__review-container">
+        <div class="stars stars--4">
+          <div class="stars__star"></div> 
+          <div class="stars__star"></div> 
+          <div class="stars__star"></div> 
+          <div class="stars__star"></div> 
+          <div class="stars__star"></div>
+        </div> 
+        <div class="card__review-count">Отзывов: 5</div>
+      </div>
+      <div class="card__review-container">
```

Comment:
```suggestion
      <div class="card__price-container">
```
<img width="1440" alt="Screenshot 2022-07-16 at 09 59 20" src="https://user-images.githubusercontent.com/94289038/179344121-0fc058ca-b200-41d1-99c0-bcf5849b2c44.png">